airflow/serialization/serialized_objects.py

```diff
@@ -710,6 +721,34 @@ def _serialize_operator_extra_links(cls, operator_extra_links: Iterable[BaseOper
 
         return serialize_operator_extra_links
 
+    @classmethod
+    def _deserialize_operator_params(cls, encoded_op_params: Dict) -> Dict[str, Param]:
+        """Deserialize Params dict of a operator"""
+        op_params = {}
+        for k, v in encoded_op_params.items():
+            param_class = import_string(v['__class'])
+            del v['__class']
+            op_params[k] = param_class(**v)
+
+        return ParamsDict(op_params)
+
+    @classmethod
+    def _serialize_operator_params(cls, op_params: ParamsDict):
+        """Serialize Params dict of a operator"""
+        serialized_params = {}
+        for k, v in op_params.items():
+            # To support backward compatability with some operators, we would let str params
+            # serialized as well, after converting them into a shallow Param object.
+            if v.__class__.__name__ == 'str':
```

Comment:
nit: Can we do the following instead?

```suggestion
            if isintance(v, 'str'):
```